File: class26/java/AP2611-CurriculoProgramador.java
import java.util.Scanner;
public class CurriculoProgramador {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("=== MINI CURRÍCULO DE PROGRAMADOR ===");
        System.out.print("Nome: "); String nome = sc.nextLine();
        System.out.print("Principal linguagem: "); String ling = sc.nextLine();
        System.out.print("Anos de experiência: "); String anos = sc.nextLine();
        System.out.println("\n" + nome + " | " + ling + " | " + anos + " anos");
        System.out.println("Habilidades: Lógica, Algoritmos, Estruturas de Dados");
        System.out.println("\"Qualquer um pode programar. Basta começar.\"");
        sc.close();
    }
}

File: class26/cpp/AP2611-CurriculoProgramador.cpp
#include <iostream>
#include <string>
int main() {
    std::string nome, ling, anos;
    std::cout << "=== MINI CURRÍCULO DE PROGRAMADOR ===" << std::endl;
    std::cout << "Nome: "; std::getline(std::cin, nome);
    std::cout << "Principal linguagem: "; std::getline(std::cin, ling);
    std::cout << "Anos de experiência: "; std::getline(std::cin, anos);
    std::cout << "\n" << nome << " | " << ling << " | " << anos << " anos" << std::endl;
    std::cout << "Habilidades: Lógica, Algoritmos, Estruturas de Dados" << std::endl;
    std::cout << "\"Qualquer um pode programar. Basta começar.\"" << std::endl;
    return 0;
}